drivers/sensor/tdk/icm456xx/Kconfig

```diff
@@ -0,0 +1,68 @@
+# ICM45686-P ICM45686-S Six-Axis Motion Tracking device configuration options
+#
+# Copyright (c) 2025 TDK Invensense
+# Copyright (c) 2022 Esco Medical ApS
+# Copyright (c) 2020 TDK Invensense
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
As this is a fresh new file please only keep: 
```
/*
 * Copyright (c) 2025 TDK Invensense
 *
 * SPDX-License-Identifier: Apache-2.0
 */
```
The Esco Medical Aps doesn't provide inputs for this new Newport driver. I also double check with SCM team which is used to Apache license.

Reply:
Fixed